package.json

```diff
@@ -21,6 +21,7 @@
     "@shelf/jest-dynamodb": "^1.7.0",
     "dynamoose": "^2.5.0",
     "jest": "^26.6.3",
+    "superagent": "^6.1.0",
```

Comment:
On your README I would like to see the input required and output returned, then the root URL added